lib/config/fileconf.go

```diff
@@ -1873,6 +1873,12 @@ type Proxy struct {
 	//
 	//nolint:revive // Because we want this to be IdP.
 	IdP IdP `yaml:"idp,omitempty"`
+
+	UiConfig UiConfig `yaml:"ui,omitempty"`
+}
+
+type UiConfig struct {
+	ScrollbackLength int `yaml:"scrollback_length,omitempty"`
```

Comment:
```suggestion
	ScrollbackLines int `yaml:"scrollback_lines,omitempty"`
```

Lines is preferred over length, because it's clear that this is a number of lines and not a number of bytes (for example)